### ISSUE TITLE
Consider adding CHANGELOG

### ISSUE THREAD

**Issue by nikwen** [wadey/node-microtime#62]
Please add a `CHANGELOG` file which tells us what to expect in major version bumps. Going through individual commits is a pain in the ass.

Apart from that, thank you very much for coding this great little library. :blush:

**Comment by nikwen** [comment]
I see that a similar request was filed when the version bump to v2 happened: https://github.com/wadey/node-microtime/issues/35

**Comment by wadey** [comment]
You are correct, we really should have this file. I will work on this. Thanks!

**Comment by nikwen** [comment]
Thank you! :)
